src/Extensions/StringExtensions.cs

```diff
@@ -59,5 +60,10 @@ private static string Capitalize(this string s)
         {
             return CultureInfo.CurrentCulture.TextInfo.ToTitleCase(s.ToLower());
         }
+
+        public static string GetStepValue(this string stepText)
```

Comment:
I don't like string extensions, and would like to get rid of it as much as possible.

The problem is that `GetStepValue` is a very niche method that gets attached to all `string`. In this case, since there are only two use cases, you can consider moving this 1 line method into the classes.

Reply:
done